exchange/bidder.go

```diff
@@ -412,3 +425,34 @@ type httpCallInfo struct {
 	response *adapters.ResponseData
 	err      error
 }
+
+// This function adds an httptrace.ClientTrace object to the context so, if connection with the bidder
+// endpoint is established, we can keep track of whether the connection was newly created, reused, and
+// even know if it was left idle and for how much time.
```

Comment:
Might want to update the comment to remove the idle references.

Reply:
Done